nerfstudio/data/dataparsers/nerfstudio_dataparser.py

```diff
@@ -258,21 +268,22 @@ def _generate_dataparser_outputs(self, split="train"):
         )
         return dataparser_outputs
 
-    def _get_fname(self, filepath: PurePath, downsample_folder_prefix="images_") -> Path:
+    def _get_fname(self, filepath: PurePath, data_dir: PurePath, downsample_folder_prefix="images_") -> Path:
         """Get the filename of the image file.
         downsample_folder_prefix can be used to point to auxillary image data, e.g. masks
+        data_dir: the directory of the data that contains the transform file
```

Comment:
I realize this was an existing issue, but can you add doc for the missing arguments.